src/screens/SpeakersScreen.tsx

```diff
@@ -73,15 +74,22 @@ const placeholder = require("../assets/img/DummySpeakerProfilePicture.jpeg")
 
   const SpeakersScreen = ({navigation}: NativeStackScreenProps<ParamListBase, screen_names.SPEAKERS, undefined>) => {
     return (
-        <View>
+        <SafeAreaView style={styles.container}>
         <FlatList
           data={MOCK_DATA}
           renderItem={({item}) => <SpeakerCard SpeakersName={item.SpeakersName} id={item.id} ProfilePicture={item.ProfilePicture} Content={item.Content}/>}
           keyExtractor={(item: SpeakerCardProps) => item.id}
           numColumns = {2}
         />
-        </View>
+        </SafeAreaView>
     );
   };
 
+  const styles = StyleSheet.create({
+    container: {
+      backgroundColor: colors.DROIDCONKE_WHITE,
+      padding: 5,
```

Comment:
Ive noticed some imbalance with padding 5. My bad 🙈
You can try playing around with padding 6-7. I trust your judgment and I won't ask for another change.

Reply:
Imbalance ?

Reply:
sorry, 
so I have reexamined the screen and there's no imbalance. 
My bad 🫢